ludwig/features/category_feature.py

```diff
@@ -240,7 +237,7 @@ def create_preproc_module(metadata: Dict[str, Any]) -> torch.nn.Module:
 
 
 class CategoryOutputFeature(CategoryFeatureMixin, OutputFeature):
-    metric_functions = {LOSS: None, ACCURACY: None, HITS_AT_K: None}
+    metric_functions = CategoryOutputFeatureConfig.get_output_metric_functions()
```

Comment:
I like this much better!

Reply:
This would be cleaner, but there might be some complexity with how `OutputFeature` base class shadows the class attribute, with `def _setup_metrics(self)`.

IMO, probably worth refactoring/cleaning this up separately.